### PR TITLE
Documentation: Add notices about reporting security flaws

### PR DESCRIPTION
**Pull request by amotl**
## About

This patch follows the suggestion by @Tu0Laj1 at [^1], in order to improve the guidance about reporting security flaws. ~~The `SECURITY.rst` file has been derived from [^2].~~ The `SECURITY.md` file has been derived from [^3], as suggested on the review of the patch.

[^1]: https://community.cratedb.com/t/mitigations-for-reported-vulnerability/1676/7
[^2]: https://github.com/dec0dOS/amazing-github-template/blob/main/%7B%7Bcookiecutter.repo_slug%7D%7D/docs/SECURITY.md
[^3]: https://github.com/electron/electron/blob/main/SECURITY.md

### CODE REVIEW

**Review by matriv — COMMENTED** [review]
Thank you! I think it looks good, not approving for now, I'll wait for the opinion of others.

File: README.rst [hunk 1]
```diff
@@ -104,6 +104,14 @@ contributions!
 See the `developer docs`_ and the `contribution docs`_ for more information.
 
 
+Security
+========
+
+If you think you discovered a security flaw, please report it to security@crate.io.
```
Comment (seut) [inline README.rst:110]:
```suggestion
If you think you discovered a security flaw, please report it to security@cratedb.com.
```

File: SECURITY.rst [hunk 2]
```diff
@@ -0,0 +1,26 @@
+Security Policy
+===============
+
+Reporting a Vulnerability
+-------------------------
+
+If there are any vulnerabilities in CrateDB, don’t hesitate to *report them*.
+
+1. Use the `private contact address <security@crate.io>`__, in order to
```
Comment (seut) [inline SECURITY.rst:9]:
```suggestion
1. Use the `private contact address <security@cratedb.com>`__, in order to
```

**Review by seut — COMMENTED** [review]
Why not use and mention Github's Security Advisories functionality? 
I think the policy file must be in the markdown format then, using a `.md` suffix.
I'd prefer mentioning this as the first option and contacting us via mail as the second one.
Even that I totally agree on  (responsible disclosure)[https://en.wikipedia.org/wiki/Coordinated_vulnerability_disclosure], the current phrasing sounds a bit harsh to me, could be understand like we want to hide reported vulnerability issues.
I think a mix between this text and e.g.  https://github.com/electron/electron/blob/main/SECURITY.md would be great.

File: README.rst (hunk 1 above)
Comment (amotl) [reply README.rst:110]: No sure if we are using cratedb.com for email already. I don't think so, but I will ask.

Comment (amotl) [reply README.rst:110]: @wmKvXo confirmed it. `crate.io` it still is.

**Comment by amotl** [comment]
Thanks for your feedback and suggestions. I will adjust the files and content as you suggested. If you want to bring it in early, feel free to take over.

**Review by mfussenegger — COMMENTED** [review]
I'd add the `SECURITY.md` to https://github.com/crate/.github  to have it enabled in all repositories.

And I'd also prefer the wording of https://github.com/electron/electron/blob/main/SECURITY.md

**Comment by amotl** [comment]
Hi again. I've updated the patch according to your suggestions, and will also prepare a similar one for https://github.com/crate/.github. Let me know about any kinds of wording adjustments you would like to see. 🙇

**Comment by amotl** [comment]
Other than the `SECURITY.md`, I haven't been able to discover the spot where to adjust the reporting template text. When comparing the templates below, I also like the template from Electron much better.

**Ours:** https://github.com/crate/crate-python/security/advisories/new
**Electron:** https://github.com/electron/electron/security/advisories/new

**Review by mfussenegger — APPROVED** [review]

**Pull request merged**